assembl/lib/logging.py

```diff
@@ -0,0 +1,109 @@
+from __future__ import absolute_import
+import traceback
+from threading import current_thread
+import logging
+
+from pyramid.threadlocal import get_current_request
+import structlog
+
+WORKSPACE_DIR_DEPTH = len(__file__.split('/')) - 3
+
+LOGGER = None
+
+
+def getCallingModuleName(skip=2):
```

Comment:
fair point on this one; I think it would be a mistake in the case of getLogger, which follows the stdlib pattern... and is all over the codebase. Sometimes the stdlib does not follow pep8, what to do? But I could be talked into it.